src/services/message.service.ts

```diff
@@ -31,6 +31,55 @@ export class MessageService {
     this._sendConstructedReport(report, { files: message.attachments.map((e) => e.url) });
   }
 
+  async sendTextMessage(
+    message: IMessage,
+    content: string,
+    argOptions: ITextMessageOptions
+  ): Promise<void> {
+    const options: ITextMessageOptions = {};
+    options.header = argOptions.header ? `${argOptions.header}\n` : '';
+    options.footer = argOptions.footer ? `\n${argOptions.footer}` : '';
+    options.reply = argOptions.reply || false;
+    options.delimiter = argOptions.delimiter === '*' ? '' : argOptions.delimiter;
+
+    const replyLength = options.reply ? `${message.author},  `.length : 0;
+    const templateLength = replyLength + options.header.length + options.footer.length;
+
+    let messagesToSend: string[] | null = [`${options.header}${content}${options.footer}`];
+
+    if (content.length + templateLength > Constants.MaxMessageLength) {
+      if (options.delimiter === undefined) {
```

Comment:
prefer !options.delimiter

Reply:
mb there, I was basically asleep when I finished up with it, and I thought there wasn't a way to achieve this.